en/setup/dev_env_windows.md

```diff
@@ -1,122 +1,56 @@
 # Windows Installation Instructions
 
-There are a number of ways you can set up a Windows development toolchain for PX4. These allow you to build for NuttX/Pixhawk and jMAVSim simulator targets.
+There are a number of ways you can set up a Windows development toolchain for PX4. Depending on what option you choose these allow you to build for NuttX/Pixhawk and jMAVSim simulator targets.
```

Comment:
Changed some of these introduction comments because they seem to be outdated/only apply to the old msys based toolchain.